rpc-rs/examples/request.rs

```diff
@@ -0,0 +1,64 @@
+use anyhow::{anyhow, Result};
+use clap::Parser;
+use nkeys::KeyPairType;
+use std::path::PathBuf;
+use std::sync::Arc;
+use wascap::prelude::KeyPair;
+use wasmbus_rpc::rpc_client::RpcClient;
+
+/// RpcClient test CLI for making nats request
+#[derive(Parser)]
+#[clap(version, about, long_about = None)]
+struct Args {
+    /// Nats uri. Defaults to 'nats://127.0.0.1:4222'
+    #[clap(short, long)]
+    nats: Option<String>,
+
+    /// File source for payload
+    #[clap(short, long)]
+    file: Option<PathBuf>,
+
+    /// Raw data for payload (as string)
+    #[clap(short, long)]
+    data: Option<String>,
+
+    /// Optional timeout in milliseconds
+    #[clap(short, long)]
+    timeout_ms: Option<u32>,
+
+    /// Subject (topic)
+    #[clap(value_parser)]
+    subject: String,
+}
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    //wasmbus_rpc::tracing::init_otel_tracing("request_bin", true);
```

Comment:
Is this leftover or should it be commented back in?

Reply:
I don't want to include it because it generates error messages if otel is not defined, but it's a useful placeholder in case someone does want to add it. Added an extra comment explaining when you might want to uncomment it.